sktime/forecasting/tests/test_ets.py

```diff
@@ -52,3 +60,29 @@ def test_airline_allow_multiplicative_trend():
     ]
 
     assert_array_equal(fit_result_R, fit_result)
+
+
+# Ignore infinite IC models when ignore_inf_ic is True
+def test_inf_ic_true():
+    forecaster = AutoETS(auto=True, sp=52, n_jobs=-1, ignore_inf_ic=True)
+    forecaster.fit(inf_ic_ts)
+    fitted_forecaster = forecaster._fitted_forecaster
+    # check that none of the information criteria are finite
```

Comment:
```suggestion
    # check that all of the information criteria are finite
```

Reply:
I think those and the above comments are fixed in the latest commit (called "fix comments"), unless I'm too sleepy now to tell the difference.